src/bridge/trx/index.ts

```diff
@@ -126,25 +126,27 @@ export class TronBridge extends Bridge {
     );
   }
 
-  async approve(approveData: ApproveData): Promise<TransactionResponse> {
-    const { tokenAddress, spender, owner } = approveData;
+  async approve(params: ApproveParamsDto): Promise<TransactionResponse> {
+    const { tokenAddress, spender, owner, amount } = params;
     const tokenContract = await this.getContract(tokenAddress);
+    const amountHex = amount === undefined ? MAX_AMOUNT : amountToHex(amount);
```

Comment:
This is the only case when == is better than === If you use == undefined it false on null as well